### ISSUE TITLE
JWT length is too big

### ISSUE THREAD

**Issue by fayaz07** [stackinflow/node-rest-api-starter#15]
@prudhvir3ddy we are creating cert's size as 2048
the problem here is when we create the jwt, it's size is huge. Shall we decrease the cert size to 1024 or 512

**Comment by fayaz07** [comment]
Sample jwt
```
eyJhbGciOiJSUzI1NiIsInR5cCI6IkpXVCJ9.eyJhdWQiOiJUb2tlblVzZXJzIiwic3ViIjoiYXBpX0FjY2Vzc1Rva2VuIiwiaXNzIjoiRmF5YXoiLCJlbWFpbCI6ImZheWF6ZnowN0BnbWFpbC5jb20iLCJpYXQiOjE1OTc4MjczMTgsImV4cCI6MTU5NzkxMzcxOH0.vBmOeWiy2zWW92i_ZRCta_XXq1bzwU4ln62-7c9qU6FWKn7-nRyXkU5jHcLYyfdCEuS_K_-cJ-p8sD4P4pZ-JkyvLMMLhVw438AhifaPlRWqMgoF3agSnN6MEIi7Z7_KlClvgsusiqlp5gMtdtFBYa0C8dMEUzZtCxAB_6w-nBiM5RzTTsgJEhOgApm-OnvR3Mz_ELJXKAdf8ZJI4YqsfaiZVBZ2ko72ajyv8817T7l073Owsk3vx8moKRDWzuQphP9CAq45yrdLuB_oo07o35aatOvPExXIn38xGpYPgJjLWhvv70o0r9sIBG8fuq4jv-48fb9kpx3bH3MxMzd4AQ
```

When we use pem of size 2048
jwt size varies around 542
and for 1024 pem size, jwt size is around 380

**Comment by prudhvir3ddy** [comment]
512 should do

**Comment by fayaz07** [comment]
ok, we'll update the docs accordingly

**Comment by fayaz07** [comment]
closed by #20
